<?xml version="1.0" encoding="UTF-8"?> 
<launch>
  <!-- 
    mc_to_robot_node.launch
    VERSION 0.0.0 Last Changed 2019-05-05
    Communicates over wifi to send/recv ros msgs
    If YEAR = 2019, then YR=19,
      robot (dst) addr = 192.168.1.YR
      robot (dst) port = 55YR
    MC: src_addr = 192.168.1.10, src_port = 5510
  -->
  <!-- Interface: 
    Pub: 

    Sub:
      joy (sensor_msgs/Joy)
    
    Param:
      frequency (double) 100.0; The frequency of the read loop
  -->
  <node pkg="bigrdigr_control" type="mc_to_robot_node" name="mc_to_robot_node" output="screen" respawn="true">
    <param name="frequency" value="100.0" type="double"/>
    <param name="dst_addr" value="192.168.1.19" type="str"/>
    <param name="dst_port" value="5519" type="int"/>
    <param name="src_port" value="5510" type="int"/>
    <param name="isTeleopCtrl" value="true" type="bool"/>
  </node>
</launch>
